reference/manual-upgrade.md

```diff
@@ -0,0 +1,391 @@
+## Manual Upgrade
+
+Upgrading Mayastor from a version prior to 2.0.0 needs manual intervention. Follow the steps given below.
+To get started, take an etcd snapshot. The detailed steps for taking a snapshot can be found in the etcd [documentation](https://etcd.io/docs/v3.3/op-guide/recovery/).
```

Comment:
This seems very generic. I believe we need to document the exact steps here with examples. The etcd snapshot doc link can be added for reference.

Reply:
@anupriya0703 I have updated the confluence with the steps and output to take the backup, Please check and update the same in our document. 
https://dcsw-bolt.atlassian.net/wiki/spaces/MS/pages/201883955/Mayastor+Upgrade+from+v1+to+v2